apps/frontpage/app/docs/sitemap.ts

```diff
@@ -0,0 +1,40 @@
+import { MetadataRoute } from 'next';
+import { generateDocsTree } from '../../lib/get-tree';
+import { TreeProps, docsVersions } from '@repo/utils';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  const latestVersion = docsVersions[0];
+  const tree = generateDocsTree(`content/docs/${latestVersion.id}`);
+
+  const flattenedTree: TreeProps[] = [];
+
+  const flattenTree = (tree: TreeProps[]) => {
+    tree.forEach((node) => {
+      if (node.children) {
+        flattenedTree.push(node);
+        flattenTree(node.children);
+      }
+      flattenedTree.push(node);
+    });
+  };
+
+  flattenTree(tree);
+
+  const filteredTree = flattenedTree
+    .filter((node) => node.type !== 'directory')
```

Comment:
Does this also filter out pages like https://storybook.js.org/docs/writing-stories ? Because those _should_ be indexed.

It's only the second-level and below, e.g. https://storybook.js.org/docs/writing-stories/mocking-data-and-modules, directory nodes that we won't to filter out.

Reply:
You're right. I was filtering too much. I added the level in the loop to make sure we can filter only level 2

Reply:
Now we are filtering if this is a director + if this is only level 2.